src/Features/LanguageServer/Protocol/Handler/Configuration/DidChangeConfigurationNotificationHandler_OnInitialized.cs

```diff
@@ -0,0 +1,33 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.VisualStudio.LanguageServer.Protocol;
+using Newtonsoft.Json.Linq;
+
+namespace Microsoft.CodeAnalysis.LanguageServer.Handler.Configuration
+{
+    internal partial class DidChangeConfigurationNotificationHandler
+    {
+        public async Task OnInitializedAsync(ClientCapabilities clientCapabilities, CancellationToken cancellationToken)
+        {
+            if (clientCapabilities?.Workspace?.DidChangeConfiguration?.DynamicRegistration is true)
+            {
+                await _clientLanguageServerManager.SendRequestAsync<RegistrationParams, JObject>(
```

Comment:
do we need to do this?  Or is it enough that we request configurations from the client at least once?

Reply:
Resolved as offline discussion